### PR TITLE
User input and audio stream information

### PR DESCRIPTION
**Pull request by IguanasInPyjamas**

### CODE REVIEW

File: src/inputs.rs [hunk 1]
```diff
@@ -0,0 +1,60 @@
+use gstreamer;
+use gstreamer::prelude::*;
+use glib;
+
+use crate::stream_info;
+
+pub fn seek_right(stream_info: &mut stream_info::_StreamInfo, prev_position: gstreamer::ClockTime) -> gstreamer::ClockTime {
+    let mut previous = prev_position;
+    stream_info.playbin.seek_simple(gstreamer::SeekFlags::FLUSH | gstreamer::SeekFlags::KEY_UNIT, previous + 5 * gstreamer::SECOND).expect("Failed to seek");
+    previous = previous + 5 * gstreamer::SECOND;
```
Comment (MattAG) [inline src/inputs.rs:10]: `previous += 5 * gstreamer::SECOND` to be a little neater.

File: src/inputs.rs [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+    return previous;
+}
+
+pub fn seek_left(stream_info: &mut stream_info::_StreamInfo, prev_position: gstreamer::ClockTime) -> gstreamer::ClockTime {
+    let mut previous = prev_position;
+    stream_info.playbin.seek_simple(gstreamer::SeekFlags::FLUSH | gstreamer::SeekFlags::KEY_UNIT, prev_position - 5 * gstreamer::SECOND).expect("Failed to seek");
+    previous = previous - 5 * gstreamer::SECOND;
```
Comment (MattAG) [inline src/inputs.rs:17]: `-=` as with above

File: src/main.rs [hunk 3]
```diff
@@ -3,100 +3,152 @@ use std::io::Write;
...
+            let tags = playbin.emit("get-audio-tags", &[&idx]).unwrap().unwrap();
+            let tags = tags.get::<gstreamer::TagList>().expect("tags").unwrap();
+
+            if let Some(artist) = tags.get::<gstreamer::tags::LanguageCode>() {
+                println!("  Language changed to : {}", artist.get().unwrap());
+            }
+
+            None
+        })
+        .unwrap();
```
Comment (MattAG) [inline src/main.rs:88]: Might be able refactor this into a register events (or something like that) file.

**Review by MattAG — APPROVED** [review]
Cool stuff

File: src/main.rs (hunk 3 above)
Comment (IguanasInPyjamas) [reply src/main.rs:88]: great idea, we could also put handle message in there too

**Pull request merged**